lib/schematype.js

```diff
@@ -900,7 +900,7 @@ function asyncValidate(validator, scope, value, props, cb) {
       return;
     }
     called = true;
-    if (typeof returnVal === 'boolean') {
+    if (typeof customMsg === 'boolean') {
       return;
     }
```

Comment:
Good spot! After review the whole code, I find this code redundant. This if block should be totally removed. Can you do that?

It worked before because we used `var returnVal` and now we use `const returnVal` so it fails.

When we used `var returnVal` in the same line of code, it' s always get undefined so that's why this if check is unnecessary. Can you delete these 3 lines?